plugins/packages/smtp_server/lib/manifest.json

```diff
@@ -0,0 +1,79 @@
+
+{
+  "$schema": "https://raw.githubusercontent.com/ToolJet/ToolJet/develop/plugins/schemas/manifest.schema.json",
+  "title": "SMTP Server datasource",
+  "description": "A schema defining SMTP Server datasource",
+  "type": "api",
+  "source": {
+    "name": "SMTP Server",
+    "kind": "smtp_server",
+    "exposedVariables": {
+      "isLoading": false,
+      "data": {},
+      "rawData": {}
+    },
+    "options": {
+      "host": {
+        "type": "string"
```

Comment:
Type is no longer used, you can remove them and keep the keys which needs to be encrypted.
```
      "user": {
        "encrypted":true
      },
      "password": {
        "encrypted": true
      }
```